### DISCUSSION TITLE
User unable to log into their account. (Forwarding from help forum)

### DISCUSSION THREAD

**Discussion by msmcculley** [ppy/osu#29334] [Q&A] [answered]
Coming from https://osu.ppy.sh/community/forums/topics/1959903?n=1. 

From the post (translated to english): 

Description of my problem: well, to understand what's going on I'll tell you a little of my story. I started getting into OS videos two months ago, I decided to download the game. When I opened the client and downloaded the game, it started updating, and then gave an error. I gave up and played like that, on the old version. And then I decided to say laser. I thought if it's a new version, the account will log in and I'll be able to play and farm PP, but it showed a different error, I allowed the application to access the Internet in both my antiviruses and firewalls, but the game still shows an error when logging into the account. What should I do?
[b]Error: connection not established because The destination computer rejected the connection request.(127.0.0.1:8892)

The network.log associated with the issue: 

[b]Error: connection not established because The destination computer rejected the connection request.(127.0.0.1:8892) .OSU Version:Laser[/b]: Request to https://auth.ppy.sh/updates/get failed with System.Net.Http.HttpRequestException: A connection could not be made because the destination computer rejected the connection request.(127.0.0.1:8892)
2024-08-07 07:34:30 [verbose]: ---> System.Net.Sockets.SocketException (10061): A connection could not be made because the destination computer rejected the connection request.
2024-08-07 07:34:30 [verbose]: at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowExcep.

Some of the things I've attempted to do was asking them to allow osu! through their firewall, updating OS, .net framework, and even repairing the .net framework. 

I'm out of ideas, so I figured I'd post here.

Thanks for the help.

**Comment by peppy** [comment]
Looks like an issue with user's proxy configuration. Can't really help much without knowing more about the proxy configuration/software.

**Reply by bdach** [reply, marked as answer]
User resolved it on their end: https://osu.ppy.sh/community/forums/posts/9631594
